psoc/grobot_workspace/grobot_module_controller.cydsn/display.c

```diff
@@ -0,0 +1,8 @@
+#include "display.h"
```

Comment:
You need to rebase onto the current PSoC branch. If you're unsure how to do that, hit me up on Slack. FYI, that's why there are all these changes to files like this one that you didn't actually touch for the DHT stuff.